src/components/layout/layout.js

```diff
@@ -128,6 +129,10 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
             Navigation = <NavCareers />
             FooterNav = <Footer no_language={true} />
             break
+        case 'p2p':
+            Navigation = <NavP2P />
+            FooterNav = <Copyright />
```

Comment:
I think the copyright should have a white color fill or maybe try using the `copyrigt.svg`  and change the fill color using styled component